styles/main.css

```diff
@@ -101,13 +101,13 @@ Im not sure why this doesn't work?*/
     height: 50px;
     background-size:contain;
     
+    position: absolute;
     top: -10px; 
     right: -7px;
-    float:right;
-    z-index: 1;
     
 }
 .mugshot{
+    position: relative;

```

Comment:
Setting `position` here does two things:
1) Makes it so the `top` property below actually has an effect
2) Makes it so that `.mugshot` is the nearest non-statically positioned element to `.paperclip` (so that `.paperclip` can be positioned relative to `.mugshot`